barretenberg/cpp/src/barretenberg/proof_system/arithmetization/arithmetization.hpp

```diff
@@ -151,16 +151,26 @@ template <typename FF_> class UltraArith {
     struct TraceBlocks {
         UltraTraceBlock pub_inputs;
         UltraTraceBlock arithmetic;
-        UltraTraceBlock sort;
+        UltraTraceBlock genperm;
         UltraTraceBlock elliptic;
         UltraTraceBlock aux;
         UltraTraceBlock lookup;
-        UltraTraceBlock main;
 
-        // TODO(https://github.com/AztecProtocol/barretenberg/issues/867): update to aux.has_ram_rom = true
-        TraceBlocks() { main.has_ram_rom = true; }
+        TraceBlocks() { aux.has_ram_rom = true; }
 
-        auto get() { return RefArray{ pub_inputs, arithmetic, sort, elliptic, aux, lookup, main }; }
+        auto get() { return RefArray{ pub_inputs, arithmetic, genperm, elliptic, aux, lookup }; }
+
+        void summarize()
```

Comment:
Thanks for this. Could you close the issue in the PR then (https://github.com/AztecProtocol/barretenberg/issues/910)?

Reply:
done